### PR TITLE
Case mgmt updates mar2023

### PR DESCRIPTION
**Pull request by beauchompers**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
N/A

## Description
Updates to the Case Management - Generic Pack from Customer Success. 

## Screenshots
Refer to video on Google Drive. (Link in Contributor form)

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [X] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [X] No

## Must have
- [ ] Tests
- [X] Documentation

### CODE REVIEW

**Comment by daryakoval** [comment]
Hi, @beauchompers!
Thanks for your contribution. 
Please merge from the master branch before I am starting the review.

File: Packs/CaseManagement-Generic/Scripts/CaseMgmtDisplayLabels/CaseMgmtDisplayLabels.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    try:
+        labels = demisto.incident().get('labels', [])
+        if labels:
+            readable = tableToMarkdown("Alert Information", labels)
+        else:
+            readable = "No labels found on Incident"
+
+        return_results(CommandResults(readable_output=readable, ignore_auto_extract=True))
+    except Exception as ex:
+        return_results(f'Failed to execute BaseScript. Error: {str(ex)}')
```
Comment (daryakoval) [inline Packs/CaseManagement-Generic/Scripts/CaseMgmtDisplayLabels/CaseMgmtDisplayLabels.py:15]:
```suggestion
        return_results(f'Failed to execute CaseMgmtDisplayLabels. Error: {str(ex)}')
```

**Review by daryakoval — CHANGES_REQUESTED** [review]

File: Packs/CaseManagement-Generic/Classifiers/classifier-Case_Management_-_Generic_Mapper.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    "feed": false,
+    "id": "Case Management - Generic Mapper",
+    "mapping": {
+        "Case": {
+            "dontMapEventToLabels": false,
+            "internalMapping": null
+        },
+        "dbot_classification_incident_type_all": {
+            "dontMapEventToLabels": false,
+            "internalMapping": null
```
Comment (dantavori) [inline Packs/CaseManagement-Generic/Classifiers/classifier-Case_Management_-_Generic_Mapper.json:12]:
```suggestion
            "internalMapping": {}
        },
        "dbot_classification_incident_type_all": {
            "dontMapEventToLabels": false,
            "internalMapping": {}
```
this should solve the build issue. @beauchompers 
1. did you write it manually or was it auto-generated?
2. what is the purpose of having a mapper without mapping data?

**Comment by daryakoval** [comment]
Hi, Please fix the following validations: 
[ERROR]: Packs/CaseManagement-Generic/Playbooks/Case_Management_-_Generic_v2.yml: [GR101] - Content item 'Case Management - Generic v2' whose from_version is '6.2.0' uses the content items: 'Case Management - Generic - Send On Call Notification' whose from_version is higher (must be equal to, or less than ..)


[ERROR]: Packs/CaseManagement-Generic/ReleaseNotes/1_4_5.md: [RN107] - No release note entry was found for the report "Case Report" in the CaseManagement-Generic pack. Please rerun the update-release-notes command without -u to generate an updated template. If you are trying to exclude an item from the release notes, please refer to the documentation found here - https://xsoar.pan.dev/docs/integrations/changelog#excluding-items

File: Packs/CaseManagement-Generic/Classifiers/classifier-Case_Management_-_Generic_Mapper.json (hunk 2 above)
Comment (beauchompers) [reply Packs/CaseManagement-Generic/Classifiers/classifier-Case_Management_-_Generic_Mapper.json]: It was auto-generated via the UI.

File: Packs/CaseManagement-Generic/Scripts/CaseMgmtDisplayLabels/CaseMgmtDisplayLabels.py (hunk 1 above)
Comment (beauchompers) [reply Packs/CaseManagement-Generic/Scripts/CaseMgmtDisplayLabels/CaseMgmtDisplayLabels.py:15]: Looks good!

**Review by daryakoval — APPROVED** [review]

**Pull request merged**